cmd/gopcomm/community_yap.gox

```diff
@@ -1,52 +1,364 @@
 import (
 	"context"
+	"time"
+	"os"
+	"io"
+	"net/http"
+	"fmt"
 
 	"github.com/goplus/community/internal/core"
+	"github.com/goplus/community/markdown"
+	"github.com/goplus/community/translation"
+	"github.com/casdoor/casdoor-go-sdk/casdoorsdk"
+	"go.uber.org/zap"
+	language "golang.org/x/text/language"
+	_ "github.com/joho/godotenv/autoload"
 )
 
 var (
 	community *core.Community
+	trans *translation.Engine
 )
 
 todo := context.TODO()
+endpoint := os.Getenv("GOP_COMMUNITY_ENDPOINT")
+domain := os.Getenv("GOP_COMMUNITY_DOMAIN")
+logger, _ := zap.NewProduction()
+defer logger.Sync()
+zlog := logger.Sugar()
+
+static "/"
+
+get "/success", ctx => {
+	ctx.yap "2xx", {}
+}
+
+get "/error", ctx => {
+	ctx.yap "4xx", {}
+}
+
+get "/failed", ctx => {
+	ctx.yap "5xx", {}
+}
 
 get "/p/:id", ctx => {
+	// Get User Info
+	var userClaim *casdoorsdk.Claims
+	token, err := ctx.Request.Cookie("token")
+	if err == nil {
+		userClaim, err = community.GetUserClaim(token.Value)
+		if err != nil {
+			zlog.Error("get user claim error:", err)
+		}
+	}
+
 	id := ctx.param("id")
 	article, _ := community.article(todo, id)
 	ctx.yap "article", {
+		"User": userClaim,
+
 		"ID":      id,
 		"Title":   article.Title,
-		"Content": article.Content,
+		"Content": article.HtmlUrl,
+		"Tags": article.Tags,
+		"Cover": article.Cover,
+		"Ctime": article.Ctime,
+		// "User": article.User,
+	}
+}
+
+get "/getArticle/:id", ctx => {
```

Comment:
it's duplicate with  `get /p/:id` ? Also, it's better to use unified style for router，